distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -995,6 +951,82 @@ requiredAttributes=["/\=","/admin\={http://schemas.xmlsoap.org/ws/2005/05/identi
 whiteListContexts=["/services/SecurityTokenService","/services/internal/metrics","/services/saml","/proxy","/services/csw"]
 ----
 
+==== Configuring Global System Properties
+The `system.properties` file found under `$DDF_HOME/etc` contains properties that will be made available through the system properties on startup. After changing any of these properties you will need to restart the system. If you change the 'hostname' property you will also need to configure the certs as described in Configuring {branding} with New Certificates.
```

Comment:
Using "you" doesn't sound appropriate. @ricklarsen, what's our standard on this?